plugins/modules/grafana_organization.py

```diff
@@ -131,7 +132,11 @@ def _send_request(self, url, data=None, headers=None, method="GET"):
             self._module.fail_json(failed=True, msg="Permission Denied")
         elif status_code == 200:
             return self._module.from_json(resp.read())
-        self._module.fail_json(failed=True, msg="Grafana Org API answered with HTTP %d" % status_code, body=self._module.from_json(resp.read()))
+        
+        if resp is None:
+            self._module.fail_json(failed=True, msg="Cannot connect to API Grafana %s" % info['msg'] , status=status_code, url=info['url'])
```

Comment:
I recall that in some cases where the queried object is not found, Grafana answer with 404 Not found, which is why the returned value is None.

It isn't perfect because when you have a 404 because of a bad URI it doesn't reflect the problem properly.
But 404 can have 2 meaning and this change would make it always considered an error while it is not.

Reply:
This part is useful when a status_code is -1 for example.